modules/core/04-channel/types/version.go

```diff
@@ -0,0 +1,27 @@
+package types
```

Comment:
should you merge main to remove this diff?

Reply:
I'd rather finalize on what we're doing regarding `v2` before merging in `main`